app/src/main/java/com/google/android/soundchecker/AudioEncoderDecoderActivity.kt

```diff
@@ -1049,6 +1056,12 @@ class AudioEncoderDecoderActivity : ComponentActivity() {
         mCodecStatus.value = report.toString()
     }
 
+    override fun onStop() {
+        super.onStop()
+
+        mOutputTrackCount++ // Stop the current playing audio stream
```

Comment:
How does this stop the stream?

Reply:
I think I figured it out but a comment would help.

Reply:
Added some comments.